stock_picking_return_restricted_qty/wizard/stock_picking_return.py

```diff
@@ -40,17 +40,6 @@ def _create_returns(self):
 class ReturnPickingLine(models.TransientModel):
     _inherit = "stock.return.picking.line"
 
-    @api.onchange("quantity")
-    def _onchange_quantity(self):
-        restrict_return_qty = (
-            self.move_id.picking_id.picking_type_id.restrict_return_qty
-        )
-
-        qty = self.get_returned_restricted_quantity(self.move_id)
-
-        if restrict_return_qty and self.quantity > qty:
-            raise UserError(_("Return more quantities than delivered is not allowed."))
```

Comment:
@ced-adhoc You remove here a fucntionality. Maybe in order to get user warned before validating the return should be to add an alert on the wizard form (if it is possible) or by other mean.